### ISSUE TITLE
Notes, dates and contact info on different location than in preview

### ISSUE THREAD

**Issue by sanderdatema** [piersdeseilligny/betterfountain#76]
When using these items:
```fountain
Notes: Some notes
Draft date: 26 augustus 2020
Contact:
    Contact 1
    Contact 2
```

The result in the preview is:
![Schermafbeelding 2020-08-26 om 19 23 24](https://user-images.githubusercontent.com/176252/91336181-d40ca100-e7d1-11ea-91bc-228128f16d09.png)

But the resulting PDF looks like this:
![Schermafbeelding 2020-08-26 om 19 23 19](https://user-images.githubusercontent.com/176252/91336218-dff86300-e7d1-11ea-8e78-c6c9872ea3f7.png)

I'd rather like the preview version.

**Comment by piersdeseilligny** [comment]
Yep, the consistency between the preview and the pdf definitely need some work...

**Comment by piersdeseilligny** [comment]
This will shortly be fixed by the next release of betterfountain - which can currently be tested in the discord server :)
